HaulageApplication/Haulage/CustomerPage.xaml

```diff
@@ -11,5 +11,11 @@
             TextColor="Black"
             VerticalOptions="Center" 
             HorizontalOptions="Center" />
+        <Button
+        x:Name="TrackItems"
+        Text="Track Items" 
+        SemanticProperties.Hint="Track customer items"
+        Clicked="CustomerNavigationButtonClicked"
```

Comment:
this should be renamed to TrackItemNavigationButtonClicked as to not be confused with customerNavigation

Reply:
Now named CustomerTrackItemNavigationButtonClicked